### PR TITLE
Activate and fix the tests that runs on webpack

### PR DESCRIPTION
**Pull request by sbrunner**
Based on #3455

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -835,6 +835,25 @@ $(EXTERNS_JQUERY): github_versions
...
+	PYTHONIOENCODING=UTF-8 .build/python-venv/bin/mako-render \
+		--var "partials=ngeo:src" \
+		--var "app=./beforeeach.js" $< > $@
+
+.PRECIOUS: test/spec/gmftemplatecache.js
+test/spec/gmftemplatecache.js: buildtools/templatecache-webpack.mako.js \
+		.build/glob2.timestamp \
+		.build/python-venv/bin/mako-render \
+		$(NGEO_PARTIALS_FILES) \
+		$(GMF_DIRECTIVES_PARTIALS_FILES)
```
Comment (gberaudo) [inline Makefile]: This has been renamed.

File: test/spec/all.js [hunk 2]
```diff
@@ -26,3 +28,4 @@ import './directives/filereader.spec.js';
 import './directives/scaleselector.spec.js';
 import './directives/layertree.spec.js';
 import './directives/grid.spec.js';
+import '../../contribs/gmf/test/spec/all.js';
```
Comment (gberaudo) [inline test/spec/all.js]: Why do you add a dependency on gmf files?

File: test/spec/services/printutils.spec.js [hunk 3]
```diff
@@ -2,11 +2,11 @@ goog.require('ngeo.print.Utils');
...
 describe('ngeo.print.Utils', () => {
 
-  let ngeoPrintUtils;
+  let ngeoPrintUtilsService;
 
   beforeEach(() => {
-    angular.mock.inject((_ngeoPrintUtils_) => {
-      ngeoPrintUtils = _ngeoPrintUtils_;
+    angular.mock.inject((ngeoPrintUtils) => {
+      ngeoPrintUtilsService = ngeoPrintUtils;
```
Comment (gberaudo) [inline test/spec/services/printutils.spec.js]: Is there a reason not to use anymore the underscore hack?

File: test/spec/all.js (hunk 2 above)
Comment (sbrunner) [reply test/spec/all.js]: Finally to don't have tow times Angular in the test scripts...

File: test/spec/services/printutils.spec.js (hunk 3 above)
Comment (sbrunner) [reply test/spec/services/printutils.spec.js]: This is not for removing the _ but to not overriding the import in es6 mode...

**Pull request merged**